src/jit/ee_il_dll.cpp

```diff
@@ -73,13 +73,19 @@ void __stdcall jitStartup(ICorJitHost* jitHost)
     if (jitstdout == nullptr)
     {
         int jitstdoutFd = _dup(_fileno(procstdout()));
```

Comment:
@gkhanna79 I can't get the repro from #6356 to work.  (I'm still trying to rework the explanation into something reproducible)

I'll make the change to check fileno as well.